apps/omg_watcher_info/lib/omg_watcher_info/db/eth_event.ex

```diff
@@ -175,14 +175,16 @@ defmodule OMG.WatcherInfo.DB.EthEvent do
           %{
             root_chain_txhash: binary(),
             log_index: non_neg_integer(),
-            output_pointer: {:utxo_position, Utxo.Position.t()} | {:output_id, tuple()}
+            output_pointer: {:utxo_position, Utxo.Position.t()} | {:output_id, tuple()},
+            eth_height: pos_integer()
           },
           available_event_type_t()
         ) :: {:ok, %__MODULE__{}} | {:error, Ecto.Changeset.t()} | :noop
   defp insert_exit!(
          %{
            root_chain_txhash: root_chain_txhash,
            log_index: log_index,
+           eth_height: eth_height,
```

Comment:
why is this function header so ugly formatted?

Reply:
This is the default formatter.